addon/bootstrap/routing.js

```diff
@@ -1,6 +1,10 @@
-import ApplicationRouteMixin from 'torii/routing/application-route-mixin';
-import AuthenticatedRouteMixin from 'torii/routing/authenticated-route-mixin';
-import { lookup, lookupFactory, register } from 'torii/lib/container-utils';
+import ApplicationRouteMixin from '@adopted-ember-addons/torii/routing/application-route-mixin';
+import AuthenticatedRouteMixin from '@adopted-ember-addons/torii/routing/authenticated-route-mixin';
```

Comment:
Would it be possible not to change the package name from just 'torii'? (This is a breaking change for anyone using imports).